server/routes/journeys/record-investigation/evidence-secured/view.njk

```diff
@@ -1,30 +1,36 @@
 {% from "govuk/components/character-count/macro.njk" import govukCharacterCount %}
 {% from "govuk/components/details/macro.njk" import govukDetails %}
-{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "partials/submitButton/macro.njk" import submitButton %}
```

Comment:
we don't have to import `govukDetails` for this view?

Reply:
you're right, we don't. looks like this was missed 2 months ago